### ISSUE TITLE
Is the split proportion correct?

### ISSUE THREAD

**Issue by sinaahmadi** [UniversalDependencies/UD_Kurmanji-MG#2]
Hi,
Currently, the data for Kurmanji is split into 20% train (even less) and 80% test set. Is that an error or have been done on purpose? If the latter, why?

**Comment by dan-zeman** [comment]
Yes, that is on purpose. The primary requirement was that there be at least 10,000 words of test data. This treebank is small, so the goal was barely exceeded. Then a tiny sample was cut off and labeled "train"; it was not meant as training data in the traditional sense but it was provided as a data sample for the participants of the [CoNLL 2017 shared task](http://universaldependencies.org/conll17/) where Kurmanji acted as a surprise language. See also the data split recommendations [here](https://universaldependencies.org/release_checklist.html#data-split).

Now, after the shared task, it is OK to take train + test concatenated and use it in your experiments, but if the experiments involve training and testing, then you have to perform 10-fold cross-validation.
